### PR TITLE
Fix VPNToggle a11y regression. Click is not handled when Space/Return…

### PR DESCRIPTION
**Pull request by yzen**
… are pressed or a11y action is triggered.

### CODE REVIEW

**Review by lesleyjanenorton — APPROVED** [review]
Thanks!

File: src/ui/components/VPNToggle.qml [hunk 1]
```diff
@@ -27,6 +27,8 @@ VPNButtonBase {
         return VPN.activate();
     }
 
+    onClicked: handleClick()
```
Comment (bakulf) [inline src/ui/components/VPNToggle.qml:30]: This is a problem. I removed it because `handleClick` is executed twice when the user clicks on the button.

**Review by bakulf — CHANGES_REQUESTED** [review]

File: src/ui/components/VPNToggle.qml (hunk 1 above)
Comment (bakulf) [reply src/ui/components/VPNToggle.qml:30]: Can you please check if this happens with your patch?

Comment (yzen) [reply src/ui/components/VPNToggle.qml:30]: Just tried logging inside `handleClick()` and getting only 1 log message in app output. I believe it looks good.. Is the consequence of double `handleClick` is that the toggle bounces off-on-off? If so it definitely does not happen.

**Pull request merged**